jekyll/_cci2/deployment-integrations.adoc

```diff
@@ -0,0 +1,171 @@
+= Configuring Deploys
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+Deployment at CircleCI is just the same as any other job configuration; you create a job and configure it to deploy to virtually any target. This document provides an overview of the deployment process, along with best practices and optimization strategies.
+
+toc::[]
+
+[discrete]
+== Overview
+
+Once a software application has been developed and tested, it needs to be deployed and made available for its intended audience. CircleCI can deploy to virtualy any target, and can be easily configured to integrate with other services for QA/testing, feature management, and deployment strategies such as blue-green or canary deployment. Quickly and easily customize your config to match your requirements, whether a fully automated process or elements of manual approval are required.
+//// 
+.Deployment
+image::pipeline-to-deployment.png[Deployment]
+////
+
+== Deployment: The Basics
+
+* **Deployment Job**: To deploy your application, add a <<jobs-steps#jobs-overview,job>> to your `.circleci/config.yml` file and configure the job to run the steps you require. The <<deployment-examples#,deployment examples>> document covers example config for many common deployment targets.
+
+* **Deployment Parameters**: You will need to add environment variables/SSH keys to fulfill your deployment steps and provide authorization details and parameters.
++
+Environment variables can be added to the project itself (follow links below for detailed instructions), or defined within your configuration. An example of defining environment variables _within_ your configuration can be seen below within the deploy job config for the non-orb Heroku example:
++
+```
+environment:
+      HEROKU_APP: "sleepy-refuge-55486"
+```
+** Add project-level <<env-vars#setting-an-environment-variable-in-a-project,environment variables>>
+** Add project-level <<add-ssh-key#,SSH keys>> 
+
+* **Manual Approval**: If your deployment strategy requires a manual approval step, you can include a <<workflows#holding-a-workflow-for-a-manual-approval,hold/approve>> job within your workflow. A manual approval button will then be available from the workflows map in the CircleCI UI.
+
+== Using Orbs to Simplify Deployment
+
+At CircleCI we offer packages of reusable configuration, known as <<orb-intro#,orbs>>. Orbs are available for many common deployment targets, which can help you to simplify and streamline your config. Check out the full range of available orbs in the https://circleci.com/orbs/registry/[Orbs Registry]. For simple pipelines simply invoking an orb, including a pre-configured job, and setting any required environment variables/SSH keys within your project, will get you the results you need with no further configuration. 
```

Comment:
The last sentence here is hard to follow. I think the sentiment you're trying to communicate is, "For simple deployment processes, an orb will get you the results you need with minimal configuration".

Reply:
Yes I agree there's no need for that sentence to be so complicated. Fixed.